spec/controllers/api/comments_controller_spec.rb

```diff
@@ -64,7 +71,20 @@
       end
 
       context 'when comments are not created' do
-        it 'retrieve valid status' do
+        it 'retrieve valid status and empty array of comments' do
+          get :index
+          expect(response.status).to eq(200)
+          expect(JSON.parse(response.body)).to eq([])
+        end
+      end
+
+      context 'when comments are created, but status is pending' do
+        before do
+          FactoryGirl.create(:comment, user: mobile_user, content: 'content comment 1', debate: debate)
+          FactoryGirl.create(:comment, user: mobile_user, content: 'content comment 2', debate: debate)
```

Comment:
You can safely remove `content` from comment creation.
Also you can add user creation to factory and remove it from here